src/avfoundation.cs

```diff
@@ -11294,6 +11778,10 @@ interface AVContentKeySessionDelegate
 
 		[Export ("contentKeySessionContentProtectionSessionIdentifierDidChange:")]
 		void DidChange (AVContentKeySession session);
+
+		[NoWatch, NoTV, NoMac, iOS (11,0)]
+		[Export ("contentKeySession:didUpdatePersistableContentKey:forContentKeyIdentifier:")]
+		void DidUpdatePersistableContentKey (AVContentKeySession session, NSData persistableContentKey, NSObject keyIdentifier);
```

Comment:
Look at swift name but `DidUpdate` seems correct (as the rest is implied with the parameter names)